src/io/flutter/inspector/HeapState.java

```diff
@@ -92,36 +95,40 @@ void addSample(HeapMonitor.HeapSample sample) {
   public void handleIsolatesInfo(VM vm, List<HeapMonitor.IsolateObject> isolates) {
     int current = 0;
     int total = 0;
+    int external = 0;
 
     isolateHeaps.clear();
 
     for (HeapMonitor.IsolateObject isolate : isolates) {
       isolateHeaps.put(isolate.getId(), isolate.getHeaps());
 
       for (HeapMonitor.HeapSpace heap : isolate.getHeaps()) {
-        current += heap.getUsed() + heap.getExternal();
-        total += heap.getCapacity() + heap.getExternal();
+        current += heap.getUsed();
+        total += heap.getCapacity();
+        external += heap.getExternal();
       }
     }
 
     rssBytes = vm.getJson().get("_currentRSS").getAsInt();
     heapMaxInBytes = total;
 
-    addSample(new HeapMonitor.HeapSample(current, false));
+    addSample(new HeapMonitor.HeapSample(current, external,false));
```

Comment:
run code format on this file and others to fix text formatting issues.

Reply:
Formatted all files in profiler directory